zypp/PathInfo.cc

```diff
@@ -661,13 +661,41 @@ namespace zypp
 
     ///////////////////////////////////////////////////////////////////
     //
+    //	METHOD NAME : safe_rename
+    //	METHOD TYPE : int
+    //
```

Comment:
`safe_rename` is considered to be an internal helper, so it should be defined inside an anonymous namespace and not be mentioned in the header file. (The 'oldfashioned ' METHOD comment block above (662-666) can be removed. We no longer do this for new code.) 
```
///////////////////////////////////////////////////////////////////
namespace
{
```